frontend/packages/ceph-storage-plugin/src/models.ts

```diff
@@ -16,7 +16,7 @@ export const CephClusterModel: K8sKind = {
 export const OCSServiceModel: K8sKind = {
   label: 'OCS Cluster Service',
   labelPlural: 'OCS Cluster Services',
-  apiVersion: 'v1alpha1',
+  apiVersion: 'v1',
```

Comment:
We might need to change version, inside the request data that we send while creating the cluster as well.